pkg/util/util.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	PromNamespace              = "ts_prom"
+	PromNamespace              = "promscale"
```

Comment:
This is a breaking change. We should mention during the release. @cevian

Reply:
Please add to notes here https://github.com/timescale/promscale/releases/tag/untagged-5b8262593f32177fe63d